### ISSUE TITLE
[Accessibility] Color Contrast is not respected.

### ISSUE THREAD

**Issue by andreacantone** [ditek/MidiSheetMusic-Android#21]
## Description

The contrast ratio between text and background should be at least 4.5 . Some elements don't respect the contrast ratio. 

## How to reproduce the error

1. Activate the app [Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=it&gl=US)
2. Open the app [Midi Sheet Music](https://play.google.com/store/apps/details?id=com.midisheetmusic&hl=it&gl=US)
3. Activate the registration from the Accessibility Scanner and use the Midi Sheet Music app.
4. After using the app, stop the recording.

## Screenshot

The results will show the errors found in the app. For example:

![IMG_20201118_155251](https://user-images.githubusercontent.com/25429624/99551049-b2c7e500-29bb-11eb-9772-5c538b45b3a0.png)
